docs/handbook/systems-administration.md

```diff
@@ -28,201 +28,354 @@ activities and many more are referred to as system administration.
 
 Basic system administration can be reduced to a number of common tasks:
 
-- Users and account management
-- Management of system resources
-- Installation and maintenance of software
+- [Accounts](#accounts)
+- [Software management](#software-management)
+- [System monitoring](#system-monitoring)
+- [Virtualisation](#virtualisation)
+- [Network configuration](#network-configuration)
+- [Network file systems (NFS)](#network-file-systems)
```

Comment:
These sections don't exist. If they are proposed, this could be noted.

Reply:
mentioned here:
> ..., but some links might to broken in the PR, but if all PRs are integrated then the links should work (or the links  
> can be fixed in the end should some PRs require change or not be accepted).
in particular 'System Monitoring' which will be a PR.
As I wrote, it was one long piece of text, which I had to break up into smaller PRs; so please be patient if some textual reference to other sections are missing; or links are missing